app/models/framework/entry_data.rb

```diff
@@ -5,19 +5,16 @@ class EntryData
     include ActiveModel::Attributes
     include ActiveModel::Validations
 
-    class << self
-      def new_from_params(params)
-        instance = new
-
-        params.each_pair do |param, value|
-          next unless instance.attributes.key?(param)
-
-          instance.send("#{param}=", value)
-        end
+    def initialize(entry)
+      super()
```

Comment:
I've updated the commit message to explain why 👍

Reply:
@tekin I actually meant *add a comment to the code*.

Reply:
yep, I prefer to put notes like this in the commit message as it's less likely to get removed accidentally, or go out of date because the code is changed without removing/updating the comment. A developer wondering "why?" can run 

```
git log -S "super()" app/models/framework/entry_data.rb
```

and get the answer.